### ISSUE TITLE
error while try to run docker

### ISSUE THREAD

**Issue by toren77** [Element84/robosat-jupyter-notebook#3]
Hi 
I'm getting error while trying to execute docker command

First command :

> docker build -t robosat-jupyter .
Sending build context to Docker daemon  89.07MB
Step 1/12 : FROM mapbox/robosat:latest-gpu
 ---> 1f15b0dbae48
Step 2/12 : RUN apt-get update &&     apt-get upgrade -y &&     apt-get install -y git vim
 ---> Using cache
 ---> 2e18a654aa08
Step 3/12 : RUN pip3 install jupyter
 ---> Using cache
 ---> ab440454bdba
Step 4/12 : RUN cd /tmp &&      git clone https://github.com/Element84/robosat &&       rsync -av /tmp/robosat/ /app
 ---> Using cache
 ---> 32e1b7fca654
Step 5/12 : RUN mkdir -p /app/container_mount/checkpoints/
 ---> Using cache
 ---> f697b9e099bb
Step 6/12 : RUN mkdir /app/robosat_container_files/
 ---> Using cache
 ---> 9a1ee36d0e00
Step 7/12 : COPY *.ipynb /app
 ---> Using cache
 ---> 98c5efaa74a2
Step 8/12 : COPY osm/*.pbf /app/container_mount
 ---> Using cache
 ---> 25979327c219
Step 9/12 : COPY images/* /app/images
When using COPY with more than one source file, the destination must be a directory and end with a /

Second command:

> docker run -d --runtime=nvidia -v /home/ubuntu/robosat/container_mount:/app/container_mount -v /home/ubuntu/robosat_container_files:/app/robosat_container_files -e DESIRED_ZOOM_LEVEL=19 -e PUBLIC_IP=93.172.121.253 -p 8888:8888 -p 5000:5000 -t robosat-jupyter jupyter notebook --ip=0.0.0.0 --allow-root

> Unable to find image 'robosat-jupyter:latest' locally
docker: Error response from daemon: pull access denied for robosat-jupyter, repository does not exist or may require 'docker login'.

**Comment by stark525** [comment]
Hello,

Are you building and running the docker image on the same machine? Both the `docker build` and `docker run` commands are intended to be ran from the same CUDA-enabled machine (EC2 instance).

**Comment by toren77** [comment]
Yes . Do you need any additional info ?

**Comment by stark525** [comment]
Yes, please. What AMI are you using? What’s your docker version?

**Comment by hadilou** [comment]
Make sure you are running the right docker image..Your docker image may not be robosat-jupyter. run "sudo docker images -all" to see the list of images. And also make sure the tag is latest not something else.
